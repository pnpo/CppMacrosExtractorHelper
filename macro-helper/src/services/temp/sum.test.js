import sum from './sum';

describe('sum tests suite', () => {
  
    it('add without params retun NaN', () => {

        const result = sum();

        expect(result).toMatchSnapshot();
        //expect(result).toBe('NaN')
    });

    it('add with one params', () => {

        const result = sum(1);

        expect(result).toMatchSnapshot();
        //expect(result).toBe(NaN)
    });

    it('add with params', () => {

        const result = sum(1, 1);

        //expect(result).toBe(2)
        expect(result).toMatchSnapshot();
    });
});