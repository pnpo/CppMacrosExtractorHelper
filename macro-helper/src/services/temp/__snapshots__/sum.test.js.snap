// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sum tests suite add with one params 1`] = `NaN`;

exports[`sum tests suite add with params 1`] = `2`;

exports[`sum tests suite add without params retun NaN 1`] = `NaN`;
